### ISSUE TITLE
Rename repo to org-id.guide

### ISSUE THREAD

**Issue by andylolz** [OpenDataServices/org-ids#180]
This repo doesn’t contain any organisation identifiers (org-ids) so the repo name is misleading! One alternative would be to name it org-id.guide

**Comment by BobHarper1** [comment]
Might I suggest org-id.guide to make it specific to the site? I'm always getting confused between this repo and org-id/register (also do we want to move this to the org-id organisation?) Could be a headache, too many orgs!

**Comment by andylolz** [comment]
> (also do we want to move this to the org-id organisation?) Could be a headache, too many orgs!

Something like https://github.com/org-id/site would work.

**Comment by stevieflow** [comment]
I agree to move this repo over to https://github.com/org-id - and rename accordingly

This should be done in the next development phase

**Comment by augusto-herrmann** [comment]
I agree. I was looking for a repo of org-ids and couldn't find one.

**Comment by augusto-herrmann** [comment]
Ok, I've [found it](/org-id/register/).
